backend/chainlit/config.py

```diff
@@ -58,6 +60,9 @@
 # Chainlit server address
 # chainlit_server = ""
 
+# Secret token used to authenticate users. Changing it will invalidate all existing sessions.
+secret_token = "{random_secret()}"
```

Comment:
is this going to change everytime i stop and restart the server?

Reply:
no, because the token is saved in the `config.toml` file, and then read from there.

Reply:
unless you remove it from the `config.toml` file, then it's gonna change every time you restart the server